### ISSUE TITLE
Callback not working

### ISSUE THREAD

**Issue by konqiDAM** [topkecleon/telegram-bot-bash#183]
Hi, I changed export `CALLBACK="1"` in mycommands.conf, have added a command like this `send_inline_buttons "${CHAT[ID]}" "Press Button ..." "   Button   |RANDOM-BUTTON"` and in the mycallbacks() I added in the first line this `answer_callback_query "${iBUTTON[ID]}" "Button data is: ${iBUTTON[DATA]}"`, even a simple `send_normal_message "XXXXX" "callback..."` but nothing seems to be working.

I'm missing something or does it not work?

**Comment by gnadelwartz** [comment]
I'll have a look if button callback is working as it should

**Comment by konqiDAM** [comment]
Any news? Maybe its an error on my end. The strange thins is that the callback appears on the logs 
![image](https://user-images.githubusercontent.com/44301655/140497140-b437c8ea-a17d-4632-b468-c6bfa378dde6.png)

**Comment by gnadelwartz** [comment]
Looks like an error in bot, will have a look on weekend

**Comment by konqiDAM** [comment]
Ho, I just figured it out what happened. I fucked up. 
![image](https://user-images.githubusercontent.com/44301655/140581124-1d4e4575-6046-4c25-b433-5047c0942e23.png)
I thought it was a good idea to modify the commands.sh (even there are 2000 alerts saying not to do it) and I added a line to check permissions that I wasn't remembering. Sorry for the inconvenience. 
Also I tipped you one bat with brave, if you want it try brave to receive it. cheers
